examples/c/ex_all.c

```diff
@@ -1041,9 +1041,13 @@ connection_ops(WT_CONNECTION *conn)
     /*! [Reconfigure a connection] */
 
     /*! [Get the database home directory] */
-    printf("The database home is %s\n", conn->get_home(conn));
+    printf("The database home is: %s\n", conn->get_home(conn));
     /*! [Get the database home directory] */
 
+    /*! [Get the configuration string] */ 
+    printf("The configure is: %s\n", conn->get_configuration(conn));
```

Comment:
Looks like we missed changing two instances of configure -> configuration here and in `wiredtiger.in:2493`

Reply:
hi, @clarissecheah 
please review again.
thanks.